test/integration/test_executor.py

```diff
@@ -1,47 +0,0 @@
-from datetime import datetime
```

Comment:
why do we get rid of this test?

Reply:
Because it is testing removed PandasExecutor (`e.raw_df`)